src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+
+    {person.age !== undefined ? (
+      <p className="Person__age">I am {person.age}</p>
+    ) : null}
```

Comment:
`{age && <p className="Person__age">I am {person.age}</p>}`

Reply:
> `{person.age && <p className="Person__age">I am {person.age}</p>}`

Good point, it doesn't make sense to compare with `undefined`. And I don't see any reason for `null`